### ISSUE TITLE
What's the best Optimizer to use and what default parameters go well.

### ISSUE THREAD

**Issue by viig99** [titu1994/Keras-ResNeXt#4]
According to the paper https://arxiv.org/pdf/1611.05431.pdf
"We use SGD with a mini-batch size of 256 on 8 GPUs (32
per GPU). The weight decay is 0.0001 and the momentum
is 0.9. We start from a learning rate of 0.1, and divide it by
10 for three times using the schedule in"
does adam work better ?

**Comment by titu1994** [comment]
I haven't trained a model yet, so I can't say with certainty, but if you want to get faster convergence definitely Adam would help.
